### DISCUSSION TITLE
[Bug] Crud Service and Swagger Aggregator

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#35] [Q&A] [answered]
Hi!

I'm running a project using `crud-service:5.4.2` and `swagger-aggregator:3.4.7`

When the `crud-service` returns the documentation to the `swagger-aggregator`, I receive the following error:

```
"message":"must match exactly one schema in oneOf"}],"name":"SyntaxError"},"documentationPath":"http://mongodb-reader/documentation/json","msg":"Error validating the documentation json at path <http://mongodb-reader/documentation/json>, openapi V3 json expected"
```

Reverting the `crud-service` back to `3.4.4` seems to solve the issue

**Comment by fredmaggiowski** [comment, marked as answer]
Hey, have you tried upgrading the swagger aggregator to v3.4.8 instead?

**Reply by flower-of-the-bridges** [reply]
Thanks! `3.4.8` seems to solve the issue

It would be cool to show in the console the standard used by the documentation paths, maybe I'll add it in the roadmap
